app/helpers/shows_helper.rb

```diff
@@ -8,9 +8,11 @@ module ShowsHelper
 
 	def csv_data
 		items = []
+    
 		CSV.foreach(FILE_PATH, headers: true) do |row|
 			items << row.to_h
 		end
+    
```

Comment:
you can drop this file TBH. Btw you can simplify this by writing:
`CSV.foreach(FILE_PATH, headers: true).map(&:to_h)`

Reply:
Yes, agree. I think I was using this helper in the previous PR but not anymore. I will delete it 👍